workflows/ampliseq.nf

```diff
@@ -79,6 +83,11 @@ if ( !params.enable_conda && !params.skip_taxonomy && !params.skip_qiime ) {
     run_qiime2 = true
 } else { run_qiime2 = false }
 
+// Set cutoff to use for SH assignment
+if ( params.addsh ) {
+    vsearch_cutoff = 0.985
```

Comment:
Is that value in any way variable, i.e could there be interest to adjust that? Hard-coded values like this are typically not good. There are exceptions of course.

Reply:
The reason I put it hard-coded like that is that it's required as input by vsearch_usearchglobal, but we want to keep it the same in ampliseq as SH assignments are are performed using 1.5-SHs (i.e. the user should not change the value). In theory we might want to use other levels for the SHs in the future, but then also other things need to change. I could not figure out a good way to save the value in module config files.

Reply:
alright, thats fine than.